<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbsorptionDialog</class>
 <widget class="QDialog" name="AbsorptionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NSXTool - Absorption Correction</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>9</y>
     <width>802</width>
     <height>615</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>61</width>
     <height>458</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="button_openFile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Open ILL *jpr file</string>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/openFileIcon.png</normaloff>:/resources/openFileIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_calibrateDistance">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Calibrate distance using ruler</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/IconcalibrateDistance.jpg</normaloff>:/resources/IconcalibrateDistance.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_pickCenter">
      <property name="toolTip">
       <string>Calibrate the pin position</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/IconpickCenter.jpg</normaloff>:/resources/IconpickCenter.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_measure">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Measure distance between two points</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/IconmeasureDistance.jpg</normaloff>:/resources/IconmeasureDistance.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_pickingPoints">
      <property name="toolTip">
       <string>Add point to the Hull</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/IconpickingPoints.jpg</normaloff>:/resources/IconpickingPoints.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_removingPoints">
      <property name="toolTip">
       <string>Remove existing points</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/IconremovingPoints.jpg</normaloff>:/resources/IconremovingPoints.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_triangulate">
      <property name="toolTip">
       <string>Construct convex hull from points</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../NSXQt.qrc">
        <normaloff>:/resources/Icontriangulate.jpg</normaloff>:/resources/Icontriangulate.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../NSXQt.qrc"/>
 </resources>
 <connections/>
</ui>
